packages/entities/entities-vaults/src/components/VaultConfigCard.vue

```diff
@@ -10,7 +10,20 @@
       @fetch:error="(err: any) => $emit('fetch:error', err)"
       @fetch:success="(entity: any) => $emit('fetch:success', entity)"
       @loading="(val: boolean) => $emit('loading', val)"
-    />
+    >
+      <template #config="slotProps">
+        <ConfigCardItem
+          v-for="propKey in Object.keys(getPropValue('rowValue', slotProps) || {})"
+          :key="propKey"
+          :item="{
+            key: propKey,
+            value: getPropItemValue(propKey, slotProps),
+            label: getMetadataLabel(propKey),
+            type: (propKey === 'token' || propKey === 'approle_secret_id') ? ConfigurationSchemaType.Redacted : ConfigurationSchemaType.Text,
+          }"
+        />
+      </template>
```

Comment:
We can simplify this block:
```suggestion
      <template #config="{ rowValue }">
        <ConfigCardItem
          v-for="propKey in Object.keys(rowValue)"
          :key="propKey"
          :item="{
            key: propKey,
            value: getPropValue(propKey, rowValue),
            label: getMetadataLabel(propKey),
            type: (propKey === 'token' || propKey === 'approle_secret_id') ? ConfigurationSchemaType.Redacted : ConfigurationSchemaType.Text,
          }"
        />
      </template>
```

And then remove the `getPropItemValue` function below.